vcbuild.bat

```diff
@@ -456,9 +456,8 @@ if "%config%"=="Debug" set test_args=--mode=debug %test_args%
 if "%config%"=="Release" set test_args=--mode=release %test_args%
 echo running 'cctest %cctest_args%'
 "%config%\cctest" %cctest_args%
-REM when building a static library there's no binary to run tests
-if defined enable_static goto test-v8
 call :run-python tools\test.py %test_args%
+goto test-v8
```

Comment:
That just jumps to the next line. :-)

Reply:
I don't even recall why I added this one :) Thanks agin.